rasa/core/actions/action.py

```diff
@@ -180,6 +183,14 @@ def action_for_name_or_text(
     ):
         return defaults[action_name_or_text]
 
+    extra_defaults: Dict[str, "Action"] = reduce(
+        lambda acc, actions: {**acc, **{a.name(): a for a in actions}},
+        plugin_manager().hook.extra_default_actions(domain=domain),
+        {},
+    )
+    if action_name_or_text in extra_defaults:
+        return extra_defaults[action_name_or_text]
+
```

Comment:
I would suggest being a bit more specific here about what these extra defaults are. For now, they could be `space_activation_actions` and the hook name could also be `generate_space_activation_actions`. It's probably less generic, but will give people reading these lines a better understanding whether they should care about it or not. I am worried with the generic naming people could imagine all sorts of things 😄